doc-tool/resources/_layouts/search.html

```diff
@@ -134,33 +128,39 @@ <h1>Member Results</h1>
             };
         };
 
-        var insertMember = function(member, li) {
-            var div = document.createElement("div");
+        var insertMember = function(member, li, parentLink) {
+            const div = document.createElement("div");
```

Comment:
Unfortunately, by using `const` we might make the website unreadable on some old browsers: https://caniuse.com/#feat=const. I would suggest reverting to `div`.

Reply:
Did you mean `reverting to var`? I've reverted to `var`.